src/js/setup-audio-tracks.js

```diff
@@ -35,10 +35,30 @@ function handlePlaybackMetadataLoaded(player, tech) {
   const currentAudioTrack = mediaPlayer.getCurrentTrackFor('audio');
 
   dashAudioTracks.forEach((dashTrack) => {
-    let label = dashTrack.lang;
+    let localizedLabel;
 
-    if (dashTrack.roles && dashTrack.roles.length) {
-      label += ` (${dashTrack.roles.join(', ')})`;
+    if (Array.isArray(dashTrack.labels)) {
+      for (let i = 0; i < dashTrack.labels.length; i++) {
+        if (player.language().indexOf(dashTrack.labels[i].lang.toLowerCase()) !== -1) {
```

Comment:
This causes a problem when someone provides labels without a `lang`, since it's trying to call `toLowerCase()` on something that's `undefined`. So we'll have to check if there's actually a `lang`.

```suggestion
        if (dashTrack.labels[i].lang &&
            player.language().indexOf(dashTrack.labels[i].lang.toLowerCase()) !== -1) {
```